code/src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../Styles/Button.css';
+
+const Button = ({ button, click, disabled, text }) => {
+	return (
+		<button type={button} onClick={click} disabled={disabled}>
```

Comment:
Clever to create a re-usable button component!

Reply:
Indeed!

Reply:
thank you =)